templates/kubernetes/docs/1.17/upgrading.md

```diff
@@ -183,15 +180,15 @@ example, if you are using the flannel CNI:
 juju upgrade-charm flannel
 ```
 
-<div class="p-notification--caution">
-  <p markdown="1" class="p-notification__response">
-    <span class="p-notification__status">Note:</span>
-Some services may be briefly interrupted during the upgrade process. Upgrading
-your CNI (e.g. flannel) will cause a small amount of network downtime. Upgrading
-<strong>easyrsa</strong> will not cause any downtime. The behaviour of other
-components you have added to your cluster may vary - check individual documentation
-for these charms for more information on upgrades.
-  </p>
+<div class="p-notification--caution is-inline">
+  <div markdown="1" class="p-notification__content">
+    <span class="p-notification__title">Note:</span>
+    <p class="p-navigation__message">Some services may be briefly interrupted during the upgrade process. Upgrading
```

Comment:
Not sure why, but it doesn't appear inline in the demo

Reply:
Argh - typo